core/commands/dag/dag.go

```diff
@@ -77,8 +77,8 @@ into an object of the specified format.
 		cmds.FileArg("object data", true, true, "The object to put").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("format", "f", "Format that the object will be added as.").WithDefault("dag-cbor"),
-		cmds.StringOption("input-enc", "Format that the input object will be.").WithDefault("dag-json"),
+		cmds.StringOption("store-codec", "s", "Codec that the stored object will be encoded with").WithDefault("dag-cbor"),
+		cmds.StringOption("input-codec", "i", "Codec that the input object is encoded in").WithDefault("dag-json"),
```

Comment:
Does this sound natural? I want to make it very clear, this is how data is encoded "at rest".
```suggestion
		cmds.StringOption("codec", "Codec that the stored object will be encoded with at rest in the datastore").WithDefault("dag-cbor"),
		cmds.StringOption("input-codec", "Codec that the input object is encoded in").WithDefault("dag-json"),
```